test/test_processor/test_config.py

```diff
@@ -260,6 +260,16 @@ def test_get_masks(self):
         self.config2.settings['mask'] = None
         assert self.config2.get_masks() is None
 
+        masks2 = self.config2.get_masks()
+
+        masks3 = self.config3.get_masks()
+        # Test custom mask
+        assert masks3[0][0][0] == 1.
+        assert masks3[0][0][1] == 0.
+        # Test mask_edge_pixel
```

Comment:
Are the comment lines swapped?

Reply:
I don't think so, but the custom CR mask that I am using it a bit weird, it just alternating 1s and 0s (1,0,1,0,1,0). I use such a mask as it is visually obvious